setup.py

```diff
@@ -5,14 +5,14 @@
 from setuptools import find_packages, setup
 
 
-VERSION = "0.0.1"
+VERSION = "2.0.0"
 
 setup(
     name="jaclang-jaseci",
     version=VERSION,
     packages=find_packages(include=["jaclang_jaseci", "jaclang_jaseci.*"]),
     install_requires=[
-        "jaclang",
+        "jaclang==0.7.5",
```

Comment:
We should figure out an elegant way to handle version comparabilities between jaclang and jaclang-jaseci. Maybe pluggy has some builtin way of handling it.

Reply:
Can we also relax a few of these dependencies? Are the specific versions required to support jaseci?

Reply:
yeah thats a good idea.